content/pt-br/docs/tasks/administer-cluster/coredns.md

```diff
@@ -0,0 +1,75 @@
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+<!-- steps -->
+
+## Sobre o CoreDNS
+
+[CoreDNS](https://coredns.io) é um servidor DNS flexível e extensível
+que pode servir como Kubernetes cluster DNS.
+Como o Kubernetes, o projeto CoreDNS é hospedado pelo
+{{< glossary_tooltip text="CNCF" term_id="cncf" >}}.
+
+Você pode usar o CoreDNS ao invés de kube-dns no seu cluster substituindo por
+kube-dns em um deployment existente, ou usando ferramentas como kubeadm
+que fará o deploy e o upgrade do cluster pra você.
+
+## Instalando o CoreDNS
+
+Para deployment manual, ou substituição do kube-dns, veja a documentação no
+[CoreDNS website](https://coredns.io/manual/installation/).
+
+## Migrando para CoreDNS
+
+### Atualizando um cluster existente com kubeadm
+
+No Kubernetes versão 1.21, kubeadm removeu o suporte para  `kube-dns` como uma aplicação DNS.
+Para `kubeadm` v{{< skew currentVersion >}}, o único Cluster DNS suportado é o CoreDNS.
+
+Você pode migrar para o CoreDNS quando usar o  `kubeadm` para atualizar o cluster que está usando 
+`kube-dns`. Neste caso, `kubeadm` gera a configuração do CoreDNS
+("Corefile") basedo no `kube-dns` ConfigMap, preservando a configuração para 
+stub domains, e upstream name server.
+
+## Atualizando CoreDNS
+
+Você pode verificar a versão do CoreDNS que o kubeadm instala para cada versão do Kubernetes na pagina 
+[versão do CoreDNS no Kubernetes](https://github.com/coredns/deployment/blob/master/kubernetes/CoreDNS-k8s_version.md).
+
+CoreDNS pode ser atualizado manualmente, caso você queria atualizar somente o CoreDNS
+ou usar sua própria imagem customizada.
+Aqui está um link útil [diretriz e passo a passo](https://github.com/coredns/deployment/blob/master/kubernetes/Upgrading_CoreDNS.md)
+disponivel para garantir uma atualização tranquilo.
+Certifique-se que a configuração existente do CoreDNS ("Corefile") é mantida quando atualizar o seu cluster.
+
+Se você está atualizando o seu cluster usando a ferramenta `kubeadm`, O `kubeadm`
+pode cuidar da retenção da configuração existente do CoreDNS automaticamente.
+
+
+## Ajustando o CoreDNS
+
+Quando a utilização dos recursos é uma preocupação, pode ser útil ajustar a configuração do CoreDNS. Para mais detalhes, confira [documentação para escalonar o CoreDNS](https://github.com/coredns/deployment/blob/master/kubernetes/Scaling_CoreDNS.md).
+
+## {{% heading "whatsnext" %}}
+
+Você pode configurar o [CoreDNS](https://coredns.io) para suportar muitos outros cenários de uso que o
+kube-dns, apenas por modificar a configuração do CoreDNS("Corefile").
+Para mais informações, veja a [documentação](https://coredns.io/plugins/kubernetes/)
+para o `kubernetes` CoreDNS plugin, ou leia o 
+[Entradas DNS personalizadas para Kubernetes](https://coredns.io/2017/05/08/custom-dns-entries-for-kubernetes/).
+no blog do CoreDNS.
```

Comment:
Alguns pontos aqui:
* Acredito que "caso de uso" seja mais amplamente utilizado em engenharia de software no português do que "cenário de uso".
* A ideia do primeiro parágrafo é enfatizar que o CoreDNS suporta mais casos de uso do que o kube-dns suportava, dependendo da configuração efetuada. Porém, a adição do "apenas" dá uma ideia de simplicidade que não estava presente na documentação original.
* É legal manter um espaço entre o texto e o início dos parênteses para legibilidade.
* Frases como "CoreDNS plugin" ficam mais legíveis em português quando escritas na ordem natural (plugin do CoreDNS).
* Notei que o artigo referido do blog do CoreDNS não está traduzido. Nesse caso, seria mais interessante não traduzirmos o título, já que se trata de um link externo, e colocar uma informação ao fim de que o texto encontra-se em inglês.

Adicionei as sugestões no bloco abaixo:

```suggestion
Você pode configurar o [CoreDNS](https://coredns.io) para suportar mais casos de uso do que o
kube-dns suporta modificando a configuração do CoreDNS ("Corefile").
Para mais informações, veja a [documentação](https://coredns.io/plugins/kubernetes/)
do plugin `kubernetes` do CoreDNS, ou leia o artigo
[Custom DNS Entries For Kubernetes](https://coredns.io/2017/05/08/custom-dns-entries-for-kubernetes/) (em inglês) no blog do CoreDNS.
```